app/view_objects/assessor_interface/application_forms_index_view_object.rb

```diff
@@ -5,7 +5,13 @@ class AssessorInterface::ApplicationFormsIndexViewObject
   include Pagy::Backend
 
   def initialize(params:)
-    @params = params
+    @params =
+      params.tap do |p|
```

Comment:
Could we split this out into a method so as we can describe what's happening here... e.g `def remove_cleared_autocomplete_values` or something like that?

Reply:
Yep, good idea, I've amended the commit with that.